charts/jenkins/values.yaml

```diff
@@ -533,6 +533,15 @@ controller:
     # Set a custom namespace where to deploy PrometheusRule resource
     prometheusRuleNamespace: ""
 
+  # Enable this if you're using Google Managed Prometheus
+  podMonitor:
```

Comment:
should this be `googlePodMonitor` or nested under something.
from a quick google I found an upstream resource as well that's quite similar:

https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-pod-monitor.yaml

Reply:
rename it to `googlePodMonitor ` as suggested.

[example-app-pod-monitor.yaml](https://github.com/prometheus-operator/prometheus-operator/blob/main/example/user-guides/getting-started/example-app-pod-monitor.yaml) is just an example of how to create PodMonitoring resource on GKE cluster to scrape the metrics.

This PR will create PodMonitoring resource for Jenkins so it can create the scraping config for GMP